Project/ICSharpCode.TextEditor.csproj

```diff
@@ -1,216 +1,115 @@
-    <DebugType>full</DebugType>
-    <LangVersion>latest</LangVersion>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <Optimize>True</Optimize>
-    <AllowUnsafeBlocks>False</AllowUnsafeBlocks>
-    <CheckForOverflowUnderflow>False</CheckForOverflowUnderflow>
-    <OutputPath>bin\</OutputPath>
-    <TreatWarningsAsErrors>False</TreatWarningsAsErrors>
-    <DebugType>pdbonly</DebugType>
-    <DebugSymbols>true</DebugSymbols>
-    <LangVersion>latest</LangVersion>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
+    <UseWindowsForms>true</UseWindowsForms>
   </PropertyGroup>
+
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Drawing" />
-    <Reference Include="System.Windows.Forms" />
-    <Reference Include="System.Xml" />
+    <None Remove="Resources\ActionScript.xshd" />
```

Comment:
Could replace all these removals with one `Remove="Resources\*.xshd"`.

Reply:
📝 Some of the removals are for other extensions. If a wildcard is used, it should only replace matching lines.

Reply:
This change was done via VS.
It can certainly be optimised further